src/controls/interfaces/ILogger.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/**
+ * Defines the Logger interface to define the logging format and
+ * log level used across the framework.
+ */
+export interface ILogger {
+    moduleName: string;
+
+    error(message: string): void;
+
+    info(message: string): void;
+
+    warn(message: string): void;
+
+    debug(message: string): void;
+
+    logObject(level: string, id: string, object: any, stringify?: boolean): void;
+
+    sanitize(message: string): string;
```

Comment:
Have all log messages that are risky been wrapped with a call to sanitize? e.g.
`this.log.info(`Slot value = ${this.log.sanitize(slotValue)}`

Reply:
Yes, all the log messages containing sensitive information are being wrapped with sanitize methods